reference/user_defined_functions.rst

```diff
@@ -796,6 +823,50 @@ The example below will repair an inactive shard placement of shard 12345 which i
 
     SELECT master_copy_shard_placement(12345, 'good_host', 5432, 'bad_host', 5432);
 
+master_move_shard_placement
```

Comment:
While there might be some scenarios, this function isn't generally meant to be called manually. That said, I'm not opposed to adding this in. Might be useful to note though that this is called as part of rebalance-table-shards, and its not necessary to be called directly.

Reply:
Added a sentence about that to the intro paragraph.